app/views/moderator_applications/show.html.slim

```diff
@@ -0,0 +1,38 @@
+.container-fluid
+  .row
+    .col-lg-2
+
+    main.restore-defaults.col-lg-8
+      h1
+        = link_to(@application.user.username, user_path(@application.user))
+        | 's Moderator Application
+
+      - if current_user.is_admin?
+        a.btn.btn-primary href=moderator_applications_path style='margin-right: 5px;'
+          span.fa.fa-undo Back
+      = link_to '<span class="fa fa-edit"></span>'.html_safe, edit_moderator_application_path(@application), class: 'btn btn-primary', style: 'margin-right: 5px;'
+      = link_to '<span class="fa fa-trash"></span>'.html_safe, @application, method: :delete, data: { confirm: 'Are you sure?' }, class: 'btn btn-primary'
```

Comment:
This is an ugly trick. I copy/pasted it from existing code, but I don't like it at all. This should, again, be factored out, and probably just improved upon in general.

Reply:
Also, "Are you sure?" should probably be replaced with something more specific, like "Are you sure you want to delete your moderator application?".

Reply:
Unfortunately, this is so awkward:
```ruby
render partial: 'application/button/edit', locals: { path: edit_moderator_application_path(@application) }
```
that I'm not even sure it counts as an improvement. I really think this should look more like a function call, like:
```ruby
render_edit_button edit_moderator_application(@application)
```
but I don't know how to do that, or even if it's possible.

Also, even better would be just:
```ruby
render_edit_button @application
```
which is actually *probably* possible, but I don't know how to dynamically get the edit route off of something. I bet it's possible natively with Rails considering how dynamic Ruby is. Worst case just add a `self.edit_path` function and duck type it?